source/chapters/user_interface.rst

```diff
@@ -602,9 +621,9 @@ Hotcue Controls
    RJ Ryan <rryan@mixxx.org>
    S.Brandt <s.brandt@mixxx.org>
 
-.. figure:: ../_static/Mixxx-200-Latenight-Deck-Hotcues.png
+.. figure:: ../_static/2.2/cue-controls.png
```

Comment:
this is the wrong picture, we need one of the hotcue grid here (with 8 hotcues)
![image](https://user-images.githubusercontent.com/5934199/97329042-eb6e1600-1876-11eb-9040-bae9ff9879cd.png)

Reply:
Okay. Corrected it